src/main/java/org/apache/commons/lang3/builder/HashCodeBuilder.java

```diff
@@ -352,7 +352,8 @@ public static int reflectionHashCode(final int initialNonZeroOddNumber, final in
      *             if the Object is {@code null}
      * @throws IllegalArgumentException
      *             if the number is zero or even
-     *
+     * @throws NullPointerException
+     *			   if the Object is {@code null}
```

Comment:
Indentation does not seem to match what's above. Name the parameter, not a generic type like Object.